packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -354,11 +403,21 @@ export function getInternalReactConstants(
       case HostText:
       case Fragment:
         return null;
+      case FundamentalComponent:
+        return 'Fundamental';
```

Comment:
There's more work to do to fully support APIs like fundamental and scope, but as long as they exist and are being used within Facebook– at least now they won't show up as "Unknown" in DevTools.

PS I assume we still intend to use both APIs? I haven't been following along with either TBH.

Reply:
I don't think we ever used Fundamental. It's dead code.

Reply:
Excellent. Thanks.